crates/recent_projects/src/recent_projects.rs

```diff
@@ -117,7 +115,7 @@ impl Render for RecentProjects {
 
 pub struct RecentProjectsDelegate {
     workspace: WeakView<Workspace>,
-    workspace_locations: Vec<WorkspaceLocation>,
+    workspace_locations: Vec<(WorkspaceId, WorkspaceLocation)>,
```

Comment:
NIT: those seem to be not locations anymore, but just the `workspaces`?

Reply:
Makes sense, just a little bit confused with the workspace property above which is a Weakview